packetbeat/protos/dns/dns.go

```diff
@@ -378,8 +292,8 @@ func (dns *Dns) receivedDnsResponse(tuple *DnsTuple, msg *DnsMessage) {
 	if trans == nil {
 		trans = newTransaction(msg.Ts, tuple.Reverse(), common.CmdlineTuple{
 			Src: msg.CmdlineTuple.Dst, Dst: msg.CmdlineTuple.Src})
-		trans.Notes = append(trans.Notes, OrphanedResponseMsg)
-		logp.Debug("dns", OrphanedResponseMsg+" %s", tuple)
+		trans.Notes = append(trans.Notes, OrphanedResponse.Error())
+		logp.Debug("dns", OrphanedResponse.Error()+" %s", tuple)
```

Comment:
try to not pass a generated format string as first parameter to printf like functions. Better:

    debug("%s %s", OrphanedResponse, tuple)

where

    var debug = logp.MakeDebug("dns")